models/index.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const process = require('process');
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require(__dirname + '/../config/config.json')[env];
```

Comment:
The `config` object is being imported from `config.json`, which currently has incorrect environment variable handling. Ensure that the environment variables are accessed correctly in `config.json` to avoid issues with database configuration.